config/config.go

```diff
@@ -232,8 +232,19 @@ func (c DatabaseConfig) GetHostname() string {
 
 type PostgreSQLConfig struct {
 	DatabaseConfig
-	SSL     bool   `json:"ssl,omitempty"`
-	SSLMode string `json:"sslMode,omitempty"`
+	SSL              bool                    `json:"ssl,omitempty"`
+	SSLMode          string                  `json:"sslMode,omitempty"`
+	CaCertSecret     apiv1.SecretKeySelector `json:"caCertSecret,omitempty"`
+	ServerCertSecret apiv1.SecretKeySelector `json:"serverCertSecret,omitempty"`
```

Comment:
This field and the next are mapped to the `sslcert` and `sslkey` options in the PostgreSQL connection options. The PostgreSQL documentation [defines ](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS)them as client certificate and key filename, so naming them server cert and server key is incorrect.

Reply:
@vladlosev changed the  ServerCertSecret and ServerKeySecret as ClientCertSecret and ClientCertSecret and will change respective helm chart as well.

Reply:
cc: @sarabala1979  @rohankmr414